stream.js

```diff
@@ -6,7 +6,10 @@ module.exports = WebSocketStream
 
 function WebSocketStream(target, protocols, options) {
   var stream, socket
-  var socketWrite = process.title === 'browser' ? socketWriteBrowser : socketWriteNode
+
+  var isBrowser = process.title === 'browser'
+  var isNative = global.WebSocket
```

Comment:
ok, adding a !! noew

re: isBrowser, IMO it is not the same as isNative. it is very possible to be in a browser environment where native websocket implementations are missing (think of older browsers and iOS)